src/controllers/wallet.ts

```diff
@@ -101,7 +101,7 @@ export class WalletController {
   }
 
   public getMnemonic(): string {
-    return this.getData(MNEMONIC_KEY);
```

Comment:
`getData` goes through a whole path that ends up generating a random mnemonic every time so this just short-circuits that for now